oauth2client/contrib/gce.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, scope, **kwargs):
             scope: string or iterable of strings, scope(s) of the credentials
                    being requested.
         """
+        # if scope:
+        #     raise ValueError('GCE will not respect your scopes so what '
+        #                      'is the point?')
         self.scope = util.scopes_to_string(scope)
```

Comment:
Not a stray commit. Just editorializing by me to elicit comments. (Commit message is `Pretend commit but really we should.`)

I'm just worried that if we remove `scope` from the constructor it will break lots of people.